wavelink/__main__.py

```diff
@@ -256,7 +256,7 @@ async def main():
         try:
             location.mkdir(parents=True, exist_ok=False)
         except Exception as e:
-            print(f'Could make directory: "{location.absolute()}". {e}\nExiting the installer...')
+            print(f'Couldn\'t make directory: "{location.absolute()}". {e}\nExiting the installer...')
```

Comment:
```suggestion
            print(f'Could not make directory: "{location.absolute()}". {e}\nExiting the installer…')
```

Reply:
The ellipsis character looks weird IMO.
![image](https://user-images.githubusercontent.com/15076404/179217449-f73ec371-3daf-4dbc-9029-7c43befe1c1a.png)

I don't think I've ever seen it used in anything command-line related, Windows *or* Linux.